### PR TITLE
[feature][bugfix] support reloading fluentd

### LINKED ISSUE #40: Create a handler to reload
`fluentd` supports reloading. startup scripts should support it, and a reload handler should be created.

### LINKED ISSUE #44: reloading the service immediately after restart kills the service
```
> bundle exec kitchen converge without-ca-key-freebsd
> bundle exec kitchen login without-ca-key-freebsd
> sudo su
# service fluentd restart; service fluentd reload; echo $?
Stopping fluentd.
Starting fluentd.
0
# service fluentd status
fluentd is not running.
```

the debug log:

```
2017-05-26 09:41:29 +0000 [info]: reading config file path="/usr/local/etc/fluentd/fluent.conf"
2017-05-26 09:41:29 +0000 [info]: starting fluentd-0.12.14
2017-05-26 09:41:29 +0000 [info]: gem 'fluentd' version '0.12.14'
2017-05-26 09:41:29 +0000 [info]: adding match pattern="syslog.**" type="null"
2017-05-26 09:41:29 +0000 [info]: adding source type="syslog"
2017-05-26 09:41:29 +0000 [debug]: listening syslog socket on 127.0.0.1:5140 with udp
2017-05-26 09:41:34 +0000 [debug]: fluentd supervisor process get SIGTERM
2017-05-26 09:41:34 +0000 [debug]: fluentd main process get SIGTERM
2017-05-26 09:41:34 +0000 [debug]: getting start to shutdown main process
2017-05-26 09:41:34 +0000 [info]: shutting down fluentd
2017-05-26 09:41:34 +0000 [info]: process finished code=0
2017-05-26 09:41:34 +0000 [info]: reading config file path="/usr/local/etc/fluentd/fluent.conf"
2017-05-26 09:41:34 +0000 [info]: starting fluentd-0.12.14
2017-05-26 09:41:34 +0000 [info]: gem 'fluentd' version '0.12.14'
2017-05-26 09:41:34 +0000 [info]: adding match pattern="syslog.**" type="null"
2017-05-26 09:41:34 +0000 [info]: adding source type="syslog"
2017-05-26 09:41:34 +0000 [debug]: listening syslog socket on 127.0.0.1:5140 with udp
2017-05-26 09:41:34 +0000 [debug]: fluentd supervisor process get SIGHUP
2017-05-26 09:41:34 +0000 [info]: restarting
2017-05-26 09:41:34 +0000 [info]: reading config file path="/usr/local/etc/fluentd/fluent.conf"
2017-05-26 09:41:34 +0000 [debug]: fluentd main process get SIGTERM
2017-05-26 09:41:34 +0000 [debug]: getting start to shutdown main process
2017-05-26 09:41:34 +0000 [info]: shutting down fluentd
2017-05-26 09:41:34 +0000 [info]: process finished code=0
2017-05-26 09:41:34 +0000 [warn]: process died within 1 second. exit.
```

### PR DESCRIPTION
**Pull request by trombik**
fixes #44 
fixes #40 

* introduce fluentd_log_file
* remove `23` postfix in paths of `fluentd` binaries in OpenBSD

### CODE REVIEW

File: tasks/install-FreeBSD.yml [hunk 1]
```diff
@@ -4,6 +4,14 @@
   pkgng:
     name: rubygem-fluentd
     state: present
+  register: register_fluentd_rubygem_fluentd
+
+- name: Patch rc.d script
+  patch:
```
Comment (mitsururike) [inline tasks/install-FreeBSD.yml]: lineinfile is safer as the rc.d script might be modified in the other version of the package and the patch fails.

Comment (trombik) [reply tasks/install-FreeBSD.yml]: but by the time the system is deployed, we should be using our own package tree. it's a hack from the start and i would like to keep the patch for inclusion.

Comment (mitsururike) [reply tasks/install-FreeBSD.yml]: it should be written as a comment.

**Review by mitsururike — APPROVED** [review]

File: tasks/install-FreeBSD.yml (hunk 1 above)
Comment (trombik) [reply tasks/install-FreeBSD.yml]: fixed in b03e8b2

**Pull request merged**